.pre-commit-config.yaml

```
@@ -26,11 +26,12 @@ repos:
         additional_dependencies: [sphinx, toml]
 
   - repo: https://github.com/zizmorcore/zizmor-pre-commit
-    rev: v1.29.0
+    rev: v1.30.0
     hooks:
       - id: zizmor
+        args: [ --no-progress, --fix ]
```

YAMLlint (1.37.1)

[error] too many spaces inside brackets

(brackets)